apis/openstack-cinder/src/test/resources/snapshot_list_simple.json

```diff
@@ -6,8 +6,9 @@
             "created_at": "2012-11-02T16:23:27.000000",
             "display_description": "jclouds test snapshot",
             "volume_id": "ea6f70ef-2784-40b9-9d14-d7f33c507c3f",
+            "os-extended-snapshot-attributes:project_id": "cc03fd4f503f4d9c986b381b8abe6af5",
```

Comment:
Snapshot expect test case was failing in previous build. due to that i had to correct it.